packages/lesswrong/components/Layout.tsx

```diff
@@ -447,7 +447,7 @@ const Layout = ({currentUser, children, classes}: {
                     </DeferRender>
                   </StickyWrapper>
                 }
-                {isLWorAF && navigationFooterBar && <TabNavigationMenuFooter />}
+                {/* {isLWorAF && navigationFooterBar && <TabNavigationMenuFooter />} */}
```

Comment:
deliberate?

Reply:
Oli and I talked about doing this, he was in favor,  I figured comment it out and see how it is and then make permanent later, but I'm not attached to doing it now